### PR TITLE
Single quote bug

### PR DESCRIPTION
**Pull request by jeswanke**
does open-cluster-management/backlog#12879

using new temptifly 0.3.47
https://github.com/open-cluster-management/temptifly/pull/97

### CODE REVIEW

**Comment by KevinFCormier** [comment]
@jeswanke We're still in the process of setting up the Prow testing, so it's not expected that job will pass yet. Please check the Travis CI - Pull Request job instead, and once that passes and the PR is reviewed, and administrator can merge it for you.

File: src-web/components/ApplicationCreationPage/ApplicationCreationPage.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ import _ from 'lodash'
 // include monaco editor
 import TemplateEditor from 'temptifly'
 import 'temptifly/dist/styles.css'
+//import TemplateEditor from 'C:/Users/jswanke/git2/temptifly/src' //'temptifly'
```
Comment (chenz4027) [inline src-web/components/ApplicationCreationPage/ApplicationCreationPage.js:36]: Shouldn't we remove this comment?

Comment (jeswanke) [reply src-web/components/ApplicationCreationPage/ApplicationCreationPage.js:36]: yes :)

**Pull request closed without merging**